DependencyInjection/Configuration.php

```diff
@@ -48,6 +48,16 @@ public function getConfigTreeBuilder()
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('filesystem')
```

Comment:
Yeah

````
repositories:
    my_repository:
        type: doctrine-phpcr-odm
        options:
            # ...
````

Would be prefered, but then we lose configuration introspection as I think we would have to simply define an array prototype for the options..

But maybe we should change it is not the ideal configuration structure.